# HG changeset patch
# User Rik <rik@octave.org>
# Date 1381191023 25200
#      Mon Oct 07 17:10:23 2013 -0700
# Node ID 539aea0e59f432486844ed96ec5bf3b4db57301c
# Parent  28eb00d3edf449abb39c0253f93cd6fb2a0f15cb
edit.m: Remove debugging code from previous cset.

* scripts/miscellaneous/edit.m: Remove keyboard command.

diff --git a/scripts/miscellaneous/edit.m b/scripts/miscellaneous/edit.m
--- a/scripts/miscellaneous/edit.m
+++ b/scripts/miscellaneous/edit.m
@@ -303,17 +303,16 @@ function ret = edit (varargin)
         filelist(end+1) = [file ".m"];
       endif
       if (isempty (regexp (file, '\.cc$')))
         ## No ".cc" at the end of the file, add to the list.
         filelist(end+1) = [file ".cc"];
       endif
     endif
 
-    keyboard;
     ## If the file includes a path, it may be an overloaded function.
     if (! index (file, "@") && index (file, filesep))
       ## No "@" at the beginning of the file, add to the list.
       numfiles = numel (filelist);
       for n = 1:numfiles
         filelist(n+numfiles) = ["@" filelist{n}];
       endfor
     endif
